rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/test_get_dict_with_terminal_units_and_zones.py

```diff
@@ -0,0 +1,232 @@
+                                    "cooling_system": {
+                                        "id": "Cooling Coil 4",
+                                        "cooling_system_type": "FLUID_LOOP",
+                                        "chilled_water_loop": "Purchased CHW Loop 1",
+                                    },
+                                    "fan_system": {
+                                        "id": "VAV Fan System 4",
+                                        "fan_control": "VARIABLE_SPEED_DRIVE",
+                                        "supply_fans": [{"id": "Supply Fan 4"}],
+                                        "return_fans": [{"id": "Return Fan 4"}],
+                                    },
+                                },
+                            ],
+                        }
+                    ],
+                }
+            ],
+            "boilers": [
+                {
+                    "id": "Boiler 1",
+                    "loop": "Boiler Loop 1",
+                    "design_capacity": 117228.44444444445,
+                }
+            ],
+            "chillers": [
+                {
+                    "id": "Chiller 1",
+                    "cooling_loop": "Chiller Loop 1",
+                    "condensing_loop": "Condenser Loop 1",
+                }
+            ],
+            "pumps": [
+                {
+                    "id": "Boiler Pump 1",
+                    "loop_or_piping": "Boiler Loop 1",
+                    "speed_control": "FIXED_SPEED",
+                },
+                {
+                    "id": "Chiller Pump 1",
+                    "loop_or_piping": "Chiller Loop 1",
+                    "speed_control": "FIXED_SPEED",
+                },
+            ],
+            "fluid_loops": [
+                {
+                    "id": "Boiler Loop 1",
+                    "type": "HEATING",
+                    "heating_design_and_control": {
+                        "id": "DAC1",
+                        "minimum_flow_fraction": 0.25,
+                    },
+                },
+                {
+                    "id": "Chiller Loop 1",
+                    "type": "COOLING",
+                },
+                {
+                    "id": "Purchased HW Loop 1",
+                    "type": "HEATING",
+                },
+                {
+                    "id": "Purchased CHW Loop 1",
+                    "type": "COOLING",
+                },
+            ],
+            "external_fluid_source": [
+                {
+                    "id": "Purchased HW",
+                    "loop": "Purchased HW Loop 1",
+                    "type": "HOT_WATER",
+                },
+                {
+                    "id": "Purchased CHW",
+                    "loop": "Purchased CHW Loop 1",
+                    "type": "CHILLED_WATER",
+                },
+            ],
+        }
+    ],
+}
+
+
+def test_get_dict_with_terminal_units_and_zones():
```

Comment:
To match our test function naming convention, there should be another underscore after "test_".